src/events/routes/eventRoutes.ts

```diff
@@ -3,6 +3,6 @@ import { getEvents } from "../controllers/getEvents"
 
 const router = express.Router()
 
-router.get("/getEvents", getEvents);
+router.get("/Events", getEvents);
```

Comment:
Any reason for **E** being capital here ?

Reply:
changed to "e"